Web.R.tex

```diff
@@ -643,6 +646,7 @@ \subsection{Analysis}
 % It does, and ZeroNet does not have a DHT
 
 From the above we derive that ZeroNet data transfer protocol is not Byzantine fault tolerant since peer discovery can be compromised. The lack of a BFT peer discovery system exposes ZeroNet to potential censorship as trackers and peers alike can refuse to provide correct information when requesting clients that share some given sites. On the other hand, it is worth noting that, in order to function, peer discovery only needs one non-failed tracker that provides enough information to discover a sufficiently large portion of the network, which then allows for peer exchange to be viable and improve on connectivity with other peers. But this is not enough to obtain BFT peer discovery: censorship attacks are unlikely to target only a portion of trackers, especially since a list of the servers used by ZeroNet is directly obtainable from the project's source code, therefore we cannot trust any single tracker.
+%todo(leandro) - It's not clear to me if the model assumptions of 1/3 Byzantine is sufficient here or not.
```

Comment:
I am purposefully ignoring the assumption that only 1/3 of the network is Byzantine because it's very feasible to shut down all trackers. Obviously, if 2/3 of the trackers are assumed to be operational then we have no problem.

This is why I originally made a distinction between static and dynamic systems, where static systems didn't have the 1/3 limitation. But we removed that, so now this is without foundation.

Reply:
You could make that point here with a sentence. Something along the lines of:
"Since trackers are a small number of fixed servers, we argue that subverting more than 1/3 of them is feasible".

Reply:
Thanks, I will add that.